src/fields/Fields.cpp

```diff
@@ -161,15 +161,11 @@ Fields::AllocData (
     }
     int num_threads = 1;
 #ifdef AMREX_USE_OMP
-#pragma omp parallel
     {
-        num_threads = omp_get_num_threads();
+        num_threads = omp_get_max_threads();
     }
```

Comment:
Could remove the braces and de-indent the line (cannot use GH suggestions for that).